scrapy/log.py

```diff
@@ -1,19 +1,24 @@
 """
-Scrapy logging facility
-
-See documentation in docs/topics/logging.rst
+This module is kept to provide a helpful warning about its removal.
 """
-import sys
+
 import logging
 import warnings
 
-from twisted.python import log
+from twisted.python.failure import Failure
+
+from scrapy.exceptions import ScrapyDeprecationWarning
+
+logger = logging.getLogger(__name__)
+
+warnings.warn("Module `scrapy.log` has been deprecated, Scrapy relies now on "
+              "the builtin Python library for logging. Read the updated "
+              "logging entry in the documentation to learn more.",
+              ScrapyDeprecationWarning, stacklevel=2)
```

Comment:
>I think stacklevel=2 can be removed since the warning.warn() call is in the module level.

@berkerpeksag We're using that stacklevel so the warning shows the line where `scrapy.log` was imported, instead of the line where the warning is issued in `scrapy.log`.